### ISSUE TITLE
Patch to add --disable-conf

### ISSUE THREAD

**Issue by GoogleCodeExporter** [timmerk/libnfc#234]
```
From lego (http://www.libnfc.org/community/post/4242/#p4242):

I am using libnfc in embedded environment where there is no storage and no 
filesystem therefore no files either. In this environment conf.c does not make 
sense. I propose adding --disable-conf (i.e. conf would be enabled by default) 
argument to the configure script (and equivalent thing for cmake).
I could provide patch for the autoconf scripts and c code if there is interest 
in this feature. I don't know cmake, so someone else would need to do that for 
cmake.

The patch for disabling configuration can be found in my github clone 
git://github.com/leg0/libnfc.git branch disable-conf .
https://github.com/leg0/libnfc/tree/disable-conf

```

Original issue reported on code.google.com by `yob...@gmail.com` on 22 Feb 2013 at 7:26

**Comment by GoogleCodeExporter** [comment]
```
Before inclusion I've a question:

Any reason for mixing #if CONF & #ifdef CONF ?

```

Original comment by `yob...@gmail.com` on 22 Feb 2013 at 4:26

**Comment by GoogleCodeExporter** [comment]
```
No good reason really. I used #if CONF myself, but then I noticed that a file I 
am modifying uses #ifdef LOG, so I used the same style as was used there, but 
must have forgotten to change the other files to match. I'll change #if CONF's 
to #ifdef CONF's as it seems that it's the style used in libnfc.

```

Original comment by `ahti.legonkov` on 22 Feb 2013 at 7:15

**Comment by GoogleCodeExporter** [comment]
```
It's ok I'll fix it on-the-fly
```

Original comment by `yob...@gmail.com` on 22 Feb 2013 at 8:07

**Comment by GoogleCodeExporter** [comment]
```
Fixed by fd6d4db5b74b
I've split --disable-conf into --disable-conffiles & --disable-envvars in a 
second commit.

```

Original comment by `yob...@gmail.com` on 22 Feb 2013 at 8:58

**Comment by GoogleCodeExporter** [comment]
Original comment by `yob...@gmail.com` on 22 Feb 2013 at 8:58
- Changed state: **Fixed**
